### PR TITLE
Add documentation for NASweb integration

### PR DESCRIPTION
**Pull request by nasWebio**
## Proposed change

Add documentation for NASweb integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/98118
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/4592
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced comprehensive documentation for integrating NASweb devices with Home Assistant.
	- Added detailed configuration instructions for setting up NASweb, including API access requirements.
	- Included visual aids and references to facilitate the integration process.

- **Documentation**
	- Enhanced metadata for the NASweb integration, including category, release version, code owners, IoT class, and platforms.

### CODE REVIEW

File: source/_integrations/nasweb.markdown [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  - switch
+ha_integration_type: hub
+---
+
+  ![](/images/integrations/nasweb/nasweb_scheme.png)
+
+NASweb combines the functions of a control panel and the ability to manage building automation. The device monitors the flow of information from sensors and programmable switches and stores settings, definitions and configured actions.\
+[MORE INFO](https://www.chomtech.pl/produkt/naswebio-multisystemowy-sterownik-automatyki-budynkowej/)
+
+## NASweb device configuration for Home Assistant
```
Comment (c0ffeeca7) [inline source/_integrations/nasweb.markdown]:
```suggestion
## Prerequisites
```

File: source/_integrations/nasweb.markdown [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
```
Comment (c0ffeeca7) [inline source/_integrations/nasweb.markdown:21]:
```suggestion

Before you can add the NASweb integration to Home Assistant, you need to create a user that has access to the NASweb API.
```

File: source/_integrations/nasweb.markdown [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+Log into device `NASweb` page:
```
Comment (c0ffeeca7) [inline source/_integrations/nasweb.markdown]:
```suggestion

To create a NASweb user with access to the NASweb API, follow thses steps:

1. To log into device `NASweb` page, enter the device IP in your local network:
   - `https://` + `Device IP` + `/nasweb` (for example: `https://192.168.117.230/nasweb`)
```

File: source/_integrations/nasweb.markdown [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+`https://` + `Device IP` + `/nasweb` (for example: `https://192.168.117.230/nasweb`)
+
```
Comment (c0ffeeca7) [inline source/_integrations/nasweb.markdown]:
```suggestion
```

File: source/_integrations/nasweb.markdown [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+Click on your user in top-right corner and select `Administrators` on pop-up user panel:
```
Comment (c0ffeeca7) [inline source/_integrations/nasweb.markdown]:
```suggestion
2. In the top-right corner, select your user and in the pop-up, select **Administrators** :
```

File: source/_integrations/nasweb.markdown [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+  ![](/images/integrations/nasweb/dashboard.png)
+
+Find user which will be used by Home Assistant to access NASweb API and click `EDIT` button:
```
Comment (c0ffeeca7) [inline source/_integrations/nasweb.markdown]:
```suggestion
3. Find the user you want to have access to the NASweb API from Home Assistant. Select **Edit**.
    - The `admin` user cannot be used for this. Select another user.
```

File: source/_integrations/nasweb.markdown [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+<div class='note warning'>
+
+User `admin` cannot be used for this purpose, select user with different username
+
+</div>
```
Comment (c0ffeeca7) [inline source/_integrations/nasweb.markdown]:
```suggestion
```

File: source/_integrations/nasweb.markdown [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+
+  ![](/images/integrations/nasweb/users.png)
+
+In `Modules` left-hand column lists available modules for this user, find `API` module and click it to move it into right-hand column (`1`), which lists modules enabled for this user, then press `APPLY` (`2`) to save changes:
```
Comment (c0ffeeca7) [inline source/_integrations/nasweb.markdown]:
```suggestion
4. In the **Modules** column, find the **API** module.
5. Select the **API** module and move it into the right-hand column (`1`), which lists modules enabled for this user. 
    - To save changes, select **Apply** (`2`).
```

File: source/_integrations/nasweb.markdown [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+
+To create a NASweb user with access to the NASweb API, follow thses steps:
+
+1. To log into device `NASweb` page, enter the device IP in your local network:
+   - `https://` + `Device IP` + `/nasweb` (for example: `https://192.168.117.230/nasweb`)
+2. In the top-right corner, select your user and in the pop-up, select **Administrators** :
+
+  ![](/images/integrations/nasweb/dashboard.png)
+
+3. Find the user you want to have access to the NASweb API from Home Assistant. Select **Edit**.
```
Comment (c0ffeeca7) [inline source/_integrations/nasweb.markdown]:
```suggestion
    ![](/images/integrations/nasweb/dashboard.png)
3. Find the user you want to have access to the NASweb API from Home Assistant. Select **Edit**.
```

File: source/_integrations/nasweb.markdown [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+    - The `admin` user cannot be used for this. Select another user.
+
+  ![](/images/integrations/nasweb/users.png)
+
```
Comment (c0ffeeca7) [inline source/_integrations/nasweb.markdown]:
```suggestion
    ![](/images/integrations/nasweb/users.png)
```

File: source/_integrations/nasweb.markdown [hunk 11]
```diff
@@ -0,0 +1,57 @@
...
+ha_iot_class: Local Push
+ha_domain: nasweb
+featured: true
+ha_config_flow: true
+ha_platforms:
+  - switch
+ha_integration_type: hub
+---
+
+  ![](/images/integrations/nasweb/nasweb_scheme.png)
```
Comment (c0ffeeca7) [inline source/_integrations/nasweb.markdown]:
```suggestion
```

File: source/_integrations/nasweb.markdown [hunk 12]
```diff
@@ -0,0 +1,57 @@
...
+
+NASweb combines the functions of a control panel and the ability to manage building automation. The device monitors the flow of information from sensors and programmable switches and stores settings, definitions and configured actions.\
+[MORE INFO](https://www.chomtech.pl/produkt/naswebio-multisystemowy-sterownik-automatyki-budynkowej/)
+
```
Comment (c0ffeeca7) [inline source/_integrations/nasweb.markdown:19]:
```suggestion

  ![](/images/integrations/nasweb/nasweb_scheme.png)

```

File: source/_integrations/nasweb.markdown [hunk 13]
```diff
@@ -0,0 +1,57 @@
...
+    ![](/images/integrations/nasweb/dashboard.png)
+3. Find the user you want to have access to the NASweb API from Home Assistant. Select **Edit**.
+    - The `admin` user cannot be used for this. Select another user.
+
+    ![](/images/integrations/nasweb/users.png)
+4. In the **Modules** column, find the **API** module.
+5. Select the **API** module and move it into the right-hand column (`1`), which lists modules enabled for this user. 
+    - To save changes, select **Apply** (`2`).
+
+  ![](/images/integrations/nasweb/modules.png)
```
Comment (c0ffeeca7) [inline source/_integrations/nasweb.markdown]:
```suggestion
    ![](/images/integrations/nasweb/modules.png)
```

File: source/_integrations/nasweb.markdown [hunk 14]
```diff
@@ -0,0 +1,56 @@
...
+
+## Prerequisites
+
+Before you can add the NASweb integration to Home Assistant, you need to create a user that has access to the NASweb API.
+
+To create a NASweb user with access to the NASweb API, follow thses steps:
+
+1. To log into device `NASweb` page, enter the device IP in your local network:
+   - `https://` + `Device IP` + `/nasweb` (for example: `https://192.168.117.230/nasweb`)
+2. In the top-right corner, select your user and in the pop-up, select **Administrators** :
```
Comment (c0ffeeca7) [inline source/_integrations/nasweb.markdown]:
```suggestion
2. In the top-right corner, select your user and in the pop-up, select **Administrators**
```

File: source/_integrations/nasweb.markdown [hunk 15]
```diff
@@ -0,0 +1,56 @@
...
+
+## Prerequisites
+
+Before you can add the NASweb integration to Home Assistant, you need to create a user that has access to the NASweb API.
+
+To create a NASweb user with access to the NASweb API, follow thses steps:
+
+1. To log into device `NASweb` page, enter the device IP in your local network:
+   - `https://` + `Device IP` + `/nasweb` (for example: `https://192.168.117.230/nasweb`)
+2. In the top-right corner, select your user and in the pop-up, select **Administrators**
```
Comment (c0ffeeca7) [inline source/_integrations/nasweb.markdown]:
```suggestion
2. In the top-right corner, select your user and in the pop-up, select **Administrators**.
```

File: source/_integrations/nasweb.markdown [hunk 16]
```diff
@@ -0,0 +1,56 @@
...
+
+    ![](/images/integrations/nasweb/dashboard.png)
+3. Find the user you want to have access to the NASweb API from Home Assistant. Select **Edit**.
+    - The `admin` user cannot be used for this. Select another user.
+
+    ![](/images/integrations/nasweb/users.png)
+4. In the **Modules** column, find the **API** module.
```
Comment (c0ffeeca7) [inline source/_integrations/nasweb.markdown]:
```suggestion
4. In the left **Modules** column, find the **API** module.
```

File: source/_integrations/nasweb.markdown [hunk 17]
```diff
@@ -0,0 +1,56 @@
...
+5. Select the **API** module and move it into the right-hand column (`1`), which lists modules enabled for this user.
```
Comment (c0ffeeca7) [inline source/_integrations/nasweb.markdown]:
```suggestion
5. Select the **API** module and drag it into the right-hand column (`1`), which lists modules enabled for this user. 
```

File: source/_integrations/nasweb.markdown [hunk 18]
```diff
@@ -0,0 +1,56 @@
...
+
+1. To log into device `NASweb` page, enter the device IP in your local network:
+   - `https://` + `Device IP` + `/nasweb` (for example: `https://192.168.117.230/nasweb`)
+2. In the top-right corner, select your user and in the pop-up, select **Administrators**.
+
+    ![](/images/integrations/nasweb/dashboard.png)
+3. Find the user you want to have access to the NASweb API from Home Assistant. Select **Edit**.
+    - The `admin` user cannot be used for this. Select another user.
+
+    ![](/images/integrations/nasweb/users.png)
```
Comment (c0ffeeca7) [inline source/_integrations/nasweb.markdown]:
```suggestion
2. In the top-right corner, select your user and in the pop-up, select **Administrators**.
    ![](/images/integrations/nasweb/dashboard.png)
3. Find the user you want to have access to the NASweb API from Home Assistant. Select **Edit**.
    - The `admin` user cannot be used for this. Select another user.
    ![](/images/integrations/nasweb/users.png)
```

**Comment by c0ffeeca7** [comment]
Thank you for contributing, @nasWebio :+1: 

1. I have restructured and rephrased the Prerequisites section a bit to make it more procedural. Please review it carefully to make sure it is still correct.
2. Please add image titles in the markdown `[]`.
3. In the very first paragraph, provide a short introduction explaining what this integration actually does in Home Assistant. This can start with `The **NASweb** {% term integration %} ...`

**Comment by c0ffeeca7** [comment]
I'm [drafting this PR](https://developers.home-assistant.io/docs/review-process#prs-are-being-drafted-when-changes-are-needed), as changes are needed. Please undraft it, once the changes have been implemented and the PR is ready for review again.

File: source/_integrations/nasweb.markdown [hunk 19]
```diff
@@ -0,0 +1,56 @@
...
+ha_platforms:
+  - switch
+ha_integration_type: hub
+---
+
+
+The **NASweb** {% term integration %} brings device's smart home features into **Home Assistant**, converting them into {% term entities %} that users can monitor, control, and incorporate into {% term scripts %} and {% term automations %}.
+
+NASweb combines the functions of a control panel and the ability to manage building automation. The device monitors the flow of information from sensors and programmable switches and stores settings, definitions and configured actions.\
+[MORE INFO](https://www.chomtech.pl/produkt/naswebio-multisystemowy-sterownik-automatyki-budynkowej/)
```
Comment (frenck) [inline source/_integrations/nasweb.markdown]:
```suggestion
The **NASweb** {% term integration %} brings the device's smart home features into **Home Assistant**, converting them into {% term entities %} that users can monitor, control, and incorporate into {% term scripts %} and {% term automations %}.

NASweb combines the functions of a control panel and the ability to manage building automation. The device monitors the flow of information from sensors and programmable switches and stores settings, definitions, and configured actions. [More information.](https://www.chomtech.pl/produkt/naswebio-multisystemowy-sterownik-automatyki-budynkowej/)
```

File: source/_integrations/nasweb.markdown [hunk 20]
```diff
@@ -0,0 +1,56 @@
...
+
+  ![NASweb Smart Home Features Graph](/images/integrations/nasweb/nasweb_scheme.png)
+
+## Prerequisites
+
+Before you can add the NASweb integration to Home Assistant, you need to create a user that has access to the NASweb API.
+
+To create a NASweb user with access to the NASweb API, follow these steps:
+
+1. To log into device `NASweb` page, enter the device IP in your local network:
```
Comment (frenck) [inline source/_integrations/nasweb.markdown]: That isn't code, so it shouldn't be code fenced.

```suggestion
1. To log into device **NASweb** page, enter the device IP in your local network:
```

File: source/_integrations/nasweb.markdown [hunk 21]
```diff
@@ -0,0 +1,56 @@
...
+   - `https://` + `Device IP` + `/nasweb` (for example: `https://192.168.117.230/nasweb`)
```
Comment (frenck) [inline source/_integrations/nasweb.markdown]:
```suggestion
   - `https://<Device IP address>/nasweb` (for example: `https://192.168.117.230/nasweb`)
```

File: source/_integrations/nasweb.markdown [hunk 22]
```diff
@@ -0,0 +1,56 @@
...
+    ![Administrator User Details](/images/integrations/nasweb/modules.png)
+
+{% include integrations/config_flow.md %}
+
+{% configuration_basic %}
+Host:
+  description: NASweb device address
+
+User:
+  description: Login of user with enabled `API` module
```
Comment (frenck) [inline source/_integrations/nasweb.markdown]:
```suggestion
  description: Login of user with enabled API module
```

File: source/_integrations/nasweb.markdown [hunk 23]
```diff
@@ -0,0 +1,56 @@
...
+
+Password:
+  description: Password of user with enabled `API` module
```
Comment (frenck) [inline source/_integrations/nasweb.markdown]:
```suggestion
  description: Password of user with enabled API module
```

**Review by frenck — APPROVED** [review]

**Pull request merged**